build.env

```diff
@@ -17,7 +17,7 @@
 source ./tools/shell_functions.inc
 
 go version >/dev/null 2>&1 || fail "Go is not installed or is not in \$PATH. See https://vitess.io/contributing/build-from-source for install instructions."
-goversion_min 1.20.5 || echo "Go version reported: `go version`. Version 1.20.5+ recommended. See https://vitess.io/contributing/build-from-source for install instructions."
+goversion_min 1.20.8 || echo "Go version reported: `go version`. Version 1.20.8+ recommended. See https://vitess.io/contributing/build-from-source for install instructions."
```

Comment:
Same question here as on #13934 about enforcing major version versus minor version.

Reply:
Oh this is just recommended. 👍